lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc

```diff
@@ -37,6 +37,15 @@ function dosomething_reportback_form($form, &$form_state, $entity = NULL) {
       '#required' => TRUE,
       '#default_value' => $entity->nid,
     );
+    $form['camapgin_run_nid'] = array(
+      '#type' => 'entity_autocomplete',
+      '#title' => t('Campaign Run ID'),
+      '#description' => t("The campaign run node this reportback was submitted for."),
+      '#entity_type' => 'node',
+      '#bundles' => array('campaign_run'),
+      '#required' => TRUE,
+      '#default_value' => $entity->nid,
```

Comment:
@chloealee I'm not sure it might be there if there's an error on the form and it saves the values so you don't have to refill out the entire form